src/p_setup.c

```diff
@@ -1774,6 +1781,10 @@ void P_Init (void)
   P_InitSwitchList();
   P_InitPicAnims();
   R_InitSprites(sprnames);
+  thinkers_arena = M_InitArena(128 * 1024 * 1024);
```

Comment:
How many megabytes will the memory footprint increase with this approach?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/2349/commits/fa50857a96b5276082c5b6b04ead5075ee4187da